src/pages/Register/Register.jsx

```diff
@@ -39,266 +63,303 @@ const Register = () => {
-                        <option value="Energy">Energy</option>
-                        <option value="Agriculture">Agriculture</option>
-                        <option value="Automotive">Automotive</option>
-                        <option value="Entertainment">Entertainment</option>
-                        <option value="Transportation and Logistics">Transportation and Logistics</option>
-                        <option value="Professional Services">Professional Services</option>
-                        <option value="Other">Other</option>
-                      </Select>
-                      <Text ml={1} color="red" position="absolute" top="50%" right="0" transform="translateY(-50%)">*</Text>
-                    </Box>
-                  </>
-                ) : (
-                  <>
-                    <Text mb={4}>We see you're searching for a job! Please fill out the additional information below:</Text>
-                    <Box flex={1} mr={4} position="relative">
-                      <Text fontSize="sm">First Name</Text>
-                      <Input 
-                        placeholder="First Name" 
-                        value={firstName} 
-                        onChange={(e) => setFirstName(e.target.value)} 
-                        isRequired 
-                        _hover={{ bg: colorMode === "light" ? "gray.200" : "gray.800" }}
-                      />
-                      <Text ml={1} color="red" position="absolute" bottom="10px" right="0">*</Text>
+                      <Text ml={1} color="red" position="absolute" top="20px" right="-8px" transform="translateY(-50%)">*</Text>
                     </Box>
                   </>
                 )}
-                <Button
-                  mt={4}
-                  colorScheme="teal"
-                  width="100%"
-                  onClick={handleBack}
-                  backgroundColor={colorMode === "light" ? "#000000" : "#A96CDE"}
-                >
-                  Back
-                </Button>
-                <Button
-                  mt={2}
-                  colorScheme="teal"
-                  width="100%"
-                  onClick={handleSubmit}
-                  backgroundColor={colorMode === "light" ? "#000000" : "#A96CDE"}
-                >
-                  Register
-                </Button>
-                <Text mt={2} textAlign="center">
-                  Already have an account? <Link color="teal.500" onClick={() => navigate("/")}>Sign In</Link>
-                </Text>
-              </>
+                <Flex justifyContent="space-between" mt={4}>
+                  <Button onClick={handleBack} color="#F3F3F3" bg="#A96CDE" _hover={{ bg: "#663399" }} width={["60%", "40%"]}>
+                    Back
+                  </Button>
+                  <Button onClick={handleSubmit} color="#F3F3F3" bg="#A96CDE" _hover={{ bg: "#663399" }} width={["60%", "40%"]}>
```

Comment:
We should use the utils folder to store constants for colors so we don't have to keep using bare strings